hackvote/db/migrate/20150521210620_create_events.rb

```diff
@@ -0,0 +1,9 @@
+class CreateEvents < ActiveRecord::Migration
+  def change
+    create_table :events do |t|
+      t.string :title
```

Comment:
Prefer making columns `:null => false`. This ensures that we don't get bad data in the database.